### ISSUE TITLE
Remove Satellite word from directory name

### ISSUE THREAD

**Issue by lzap** [theforeman/foreman-documentation#194]
I think we should not use either Foreman or Satellite in directory names: `doc-Installing_Satellite_Server_Disconnected`. They are mapped to URLs and also information value is zero, the shorter URL the better.

```
doc-Content_Management_Guide
doc-Deploying_on_AWS
doc-Installing_Proxy_on_Debian
doc-Installing_Proxy_on_Red_Hat
doc-Installing_Satellite_Server_Disconnected
doc-Installing_Server_on_Debian
doc-Installing_Server_on_Red_Hat
doc-Managing_Hosts
doc-Provisioning_Guide
```

In general, we should keep them as short as possible regardless of the actual book title. If you can identify and set up some naming rules, go ahead and put this into the README. @spetrosi @melcorr  @tahliar

**Comment by melcorr** [comment]
>doc-Installing_Satellite_Server_Disconnected

Agreed, and the only reason this is left is because it is a downstream only guide with no upstream application, or else it was before I left!

**Comment by tahliar** [comment]
Although all the downstream teams I've worked on have always tried to keep the directory names matching the book titles, I'm not sure if it was actually a rule or just a thing people did.

So I'm more than happy to use a shorter standard upstream, as it's easy for us to map the directory names to each other in the sync script - or even to adopt it downstream when we create new guides, e.g. doc-Deploying_on_AWS, which has the same directory name upstream and downstream even though the title is Deploying {ProjectName} on Amazon Web Services. 

(I should note that *file* names matching titles *is* a rule, or at least encouraged, in the mod docs guidance: https://redhat-documentation.github.io/modular-docs/#anchor-and-file-names Though it does say "the same as or similar to" so I think we have some wiggle room if we want to remove Satellite/Capsule from file names as well as directories.)

**Comment by lzap** [comment]
There's more now :-) `doc-Administering_Red_Hat_Satellite`

**Comment by Lennonka** [comment]
I believe this was resolved with PR #1230 and we can close it. Ack?

**Comment by lzap** [comment]
Thank you!
